### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mfrachet**
# [title]

## Description
[add description]

## Demo
Example on : [deployed story link]

## API
[api details]

```jsx
<YourComponent />
```

## Voiceover

[voiceover gif]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
   </a>
 </p>
 
-# Welcome! 👋
+# Welcome! 👋👋👋
```
Comment (gu-stav) [inline README.md:23]:
```suggestion
# Welcome! 👋👋👋👋👋
```

**Review by gu-stav — DISMISSED** [review]
Good to see you are still following :)

**Review by HichamELBSI — APPROVED** [review]

**Comment by HichamELBSI** [comment]
Congrats you've made the first commit of the stable version 😅

**Comment by mfrachet** [comment]
What an honor! Thanks 😛 . That's good to see you, hope you're doing okay! 😊

**Pull request merged**